client/gutenberg/extensions/slideshow/slideshow.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
```

Comment:
Checked that this file doesn't get imported to `view.js` 👍 

For the measure also checked that slideshow depends only on `lodash` on view side: https://github.com/Automattic/jetpack/blob/f024195cb6f208603c6da239431c5c5bc481f3a9/extensions/blocks/slideshow/slideshow.php#L26-L28

Reply:
FYI discussion of the `lodash` dependency here: https://github.com/Automattic/wp-calypso/pull/30789